wiki/Split_large_file.md

```diff
@@ -46,6 +46,8 @@ def batch_iterator(iterator, batch_size):
         if len(batch) == batch_size:
             yield batch
             batch = []
+    if len(batch) > 0:
+        yield batch
 ```
```

Comment:
Well spotted, perhaps it was only ever tested on examples with no remainder (e.g. 50 entries in batches of 10).

Style wise I'd suggest ``if batch:`` but the fix looks good.

Reply:
> Well spotted, perhaps it was only ever tested on examples with no remainder (e.g. 50 entries in batches of 10).
> 
> Style wise I'd suggest `if batch:` but the fix looks good.

Makes sense. I had a remainder in my non-perfect fasta. 
And `if batch:` does look better.
Thanks!

Reply:
If you make that minor improvement, I'll merge this.